### ISSUE TITLE
availability of a package

### ISSUE THREAD

**Issue by NikoloAz** [bergsmat/tablet#3]
At work we have R version 4.2.3. and the package tablet doesn't work in this version of R. As i need some permissions to install new R-version and everyone shouldn't use R for some time, i wanted to ask first in which version of R is this package available? And is it okay that it doesn't work in R version 4.2.3? Thanks in advance!

**Comment by bergsmat** [comment]
Hi, and thanks for your note.  In fact, tablet works with all versions of R at least since R version 4.0.5.  I have just installed R 4.2.3 from CRAN, and successfully tested tablet 0.6.3 with this version of R.  Please try
```install.packages(repos = ‘https://cran.r-project.org’, ‘tablet’)```
or
```devtools::install_github(‘bergsmat/tablet’)```
and let me know the result.

**Comment by NikoloAz** [comment]
Hi, big thanks for your response. I actually tried to install the package in both ways already and than i come to such errors/warnings:
1. way
Warning in install.packages :
  kann nicht auf den Index für das Repository https://cran.rstudio.com/src/contrib zugreifen:
  cannot open URL 'https://cran.rstudio.com/src/contrib/PACKAGES'
Installiere Paket nach ‘C:/MVArbeit/R/Library’
(da ‘lib’ nicht spezifiziert)
Warning in install.packages :
  kann nicht auf den Index für das Repository https://cran.r-project.org/src/contrib zugreifen:
  cannot open URL 'https://cran.r-project.org/src/contrib/PACKAGES'
Warning in install.packages :
  package ‘tablet’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
Warning in install.packages :
  kann nicht auf den Index für das Repository https://cran.r-project.org/bin/windows/contrib/4.2 zugreifen:
  cannot open URL 'https://cran.r-project.org/bin/windows/contrib/4.2/PACKAGES'

2.way
Error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Connection timeout after 10006 ms

Maybe i should try to change something with the settings. I will try to figure it out... If something is new i will let you know. Thank you.

**Comment by bergsmat** [comment]
Hi again.

Sometimes the warning "package x is not available for this version of R" can be misleading, as the real problem is something more fundamental.  Seems like your system can't "see" some web resources for some reason.

I was able to install package 'tablet' for R 4.2.3 on Windows using both the methods mentioned above.  First I did default installs of all the required packages.  But to make it work, I had to update packages 'xfun', 'htmltools', and 'purrr'.  You can see my results in the attached file.  See the very end for the package versions that are working.

Please let me know how it goes.
[install-tablet.pdf](https://github.com/bergsmat/tablet/files/12715840/install-tablet.pdf)

**Comment by NikoloAz** [comment]
Hi again, 

thank you for your thorough response and your help. You are right, the problem was in proxy settings of my laptop for work and not R version. Thats why I have downloaded zip file of the package, created a project there and installed the package. I just installed few more packages to make it work (sport, yamlet, latexpdf, csv, encode and units) in the same way. Now everything works properly. 

Few months ago i was trying to do the same stuff with the package gtsummary and it was way too slow. Your package creates those tables really fast and big thanks! So the issue can be closed i think...
